packages/base-driver/lib/protocol/bidi-commands.js

```diff
@@ -0,0 +1,31 @@
+const SUBSCRIPTION_REQUEST_PARAMS = /** @type {const} */ ({
```

Comment:
would it make sense to make it a .ts file instead as we need to declare multiple types?

Reply:
all we're doing is setting these to const so IDEs know they are immutable. not sure we need typescript for that